src/output.rs

```diff
@@ -386,6 +463,16 @@ mod errors {
                 description("Output predicate failed")
                 display("{}\noutput=```{}```", msg, got)
             }
+/* Adding a single error more makes this break, using the bottom one temporarily
```

Comment:
In what way does this break?

Granted, at some point we should probably move beyond error-chain

Reply:
Out of stack space, this is a known bug in error-chain. I'm gonna look up some docs for this problem.